docs/UserGuide.md

```diff
@@ -1,186 +1,175 @@
-> The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
-
-Examples:
-
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br>
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
-
-### 2.7. Select a person : `select`
-
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
-
-> Selects the person and loads the Google search page the person at the specified `INDEX`.<br>
-> The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
-
-Examples:
-
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br>
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
-
-### 2.8. Clearing all entries : `clear`
-
-Clears all entries from the address book.<br>
-Format: `clear`
-
-### 2.9. Exiting the program : `exit`
-
-Exits the program.<br>
-Format: `exit`
-
-### 2.10. Saving the data
-
-Address book data are saved in the hard disk automatically after any command that changes the data.<br>
+> * Only full words will be matched e.g. `Milk` will not match `Milks`
+> * Tasks matching all keywords will be returned (i.e. `AND` search).
+    e.g. `Milk` will not match `Buy Milk`
+    e.g. `donuts milk` will match `Buy donuts and milk`
+> * Tasks will be matched only if its information matches all the keywords given 
+
+Examples: 
+* `find d/20/03/2017`<br>
+  Returns all tasks that have deadline due on `20/03/2017`
+* `find Buy The Milk`<br>
+  Returns only tasks containing `Buy`, `The`, and `Milk`
+
+#### Deleting a task : `kill / k`
+Deletes the specified task from the todo list.<br>
+Format: `kill INDEX`
+
+> Deletes the todo at the specified `INDEX`. 
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
+
+Examples: 
+* `kill 2`
+  Deletes the 2nd task in the most recent view of the todo list.
+  
+* `kill 1`
+  Deletes the 1st task in the most recent view of the todo list.
+ 
+#### Sort a list of task `sort / s`
+> Sort the list of tasks in a certain order (p - priority, d - deadline)
+> Format: `sort ORDERTYPE`
+
+Examples: `sort p'
+
+#### Marking a task as done : `done / d`
+Marks the specified task from the todo list as done. <br>
+Format: `done INDEX`
+
+> Marks the todo at the specified `INDEX` as done.
+ The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
+
+Examples: 
+* `done 3`<br>
+
+#### Clearing all entries : `clear / c`
+Clears all entries from the todo list.<br>
+Format: `clear`  
+* user will be prompted with clear confirmation
+
+
+
+#### Undoing the last action : `undo / u`
+Undos the last action.<br>
+Format: `undo`  
```

Comment:
More thoughts can be given to the undo function too, such as what type of commands can be undo-ed.